### ISSUE TITLE
Trying to use SSL Default Location for SWAT 

### ISSUE THREAD

**Issue by mibret** [sassoftware/python-swat#104]
Hi,

I have SWAT configured and can access CAS successfully as long as I set the CAS_CLIENT_SSL_CA_LIST environment variable. I am trying to configure this so users do not have to set this environment variable or eliminate setup of a system env var. 

I determined the default ssl location and thought adding the Viya certs to the existing cacert.pem file would work. If you look at the attached test code this is not the case. You can see setting CAS_CLIENT_SSL_CA_LIST works using the python ssl default location. Is there a way to do this?  Thanks in advance.
[swat_ssl_test.txt](https://github.com/sassoftware/python-swat/files/5665786/swat_ssl_test.txt)

**Comment by kesmit13** [comment]
Installing in the Anaconda location will likely work with HTTP(S) connections, but not binary. You might want to try copying your pem file to the following path and see if that works for you.
```
/opt/sas/viya/home/SASSecurityCertificateFramework/cacerts/trustedcerts.pem
```

**Comment by mibret** [comment]
Hi

SWAT is installed on a server with just Python. Viya is not co-located. I added the Viya root and intermediate to cacerts.pem on the server with Python.

**Comment by kesmit13** [comment]
@mibret Did this solve the problem for you?

**Comment by mibret** [comment]
Hi

The CAS controller location for trustedcerts.pem is

/opt/sas/viya/config/etc/SASSecurityCertificateFramework/cacerts

To clarify are you saying to add customers CA root to trustedcerts.pem or just adding a copy of the python cacert.pem to this folder? I tried the later and still get the error.

Note that Python with SWAT is on a remote server and the client making a remote connection to the CAS controller. In the code sample I sent, the import of the Viya root and intermediate certs to the cacert.pem on the python server addresses the issue for the binary connection. I am just trying to eliminate the setting of the environment variable.

thx

**Comment by kesmit13** [comment]
@mibret I was hoping that copying the working cacert.pem to the path I mentioned would solve the issue. Did you also change the name to trustedcerts.pem when copying it to that directory?

**Comment by mibret** [comment]
Hi

I cannot rename the trustedcerts.pem as these are used by Viya.  Viya is deployed with customer’s server certs and believe as part of the deployment are contained in these files looking as messages in the deployment log.

**Comment by kesmit13** [comment]
@mibret At the moment, that's the only location that's a possibility for SWAT looking by default. It is possible to set the certificates using an option in SWAT as well, but I don't know that will solve your problem any better than an environment variable.
```
swat.options.cas.ssl_ca_list = '/path/to/cert.pem'
```

I can look into adding Python's default ssl configuration to SWAT if it doesn't find an environment variable or option already set.

**Comment by mibret** [comment]
Hi Kevin

I will look to work with the system admins on best way to setup the SSL env var. Adding to search the Python’s ssl location would be good as customer’s security team will review this and most likely ask for that recommendation.

Thanks

**Comment by kesmit13** [comment]
@mibret I pushed a change that I think should fix your issue here. I'll have to do a release for it to be installable from PyPI / Anaconda Cloud, but you can try it out if you can install the changed file.

**Comment by kesmit13** [comment]
Fixed in v1.8.0
